lua/neogit/popups/revert/init.lua

```diff
@@ -23,6 +23,7 @@ function M.create(env)
     :group_heading("Revert")
     :action_if(not in_progress, "v", "Commit(s)", actions.commits)
     :action_if(not in_progress, "V", "Changes", actions.changes)
+    :action_if(not in_progress, "h", "Hunk", actions.hunk)
```

Comment:
```suggestion
    :action_if(((not in_progress) and env.hunk ~= nil), "h", "Hunk", actions.hunk)
```

Probably only makes sense to show the hunk action if a hunk is available for consumption.